ui/src/app/base/sagas/websockets.ts

```diff
@@ -487,14 +487,14 @@ export function* handleMessage(
         }
         if (error) {
           yield* put({
-            meta: { item },
+            meta: { item, modelPK: action.meta.modelPK },
```

Comment:
We need the `modelPK` to be present throughout the websocket saga in order to relate an error with a model, so we include it for each of the `Start`, `Success` and `Error` messages

Reply:
Updated to use `identifier`